draft-dekater-scion-controlplane.md

```diff
@@ -1262,162 +1267,165 @@ The last step of the above described core and intra-ISD propagation procedures i
+As mentioned previously, a non-core AS typically receives several PCBs representing several path segments to the core ASes of the ISD the AS belongs to. Out of these PCBs, the non-core AS selects those down path segments through which it wants to be reached, based on AS-specific selection criteria.
 
+The next step is to register the selected down segments with the Control Service of the relevant core ASes in accordance with a process called *intra-ISD path segment registration*. As a result, a core AS's Control Service contains all intra-ISD path segments registered by the non-core ASes of its ISD. In addition, each core AS Control Service also stores the preferred core path segments to other core ASes during the *core segment registration* process.
 
-## Intra-ISD Path-Segment Registration {#intra-reg}
+Both processes are described below.
 
-Every *registration period* (determined by each AS), the AS's control service selects two sets of PCBs to transform into two types of path segments:
 
-- Up-segments, which allow the infrastructure entities and endpoints in this AS to communicate with core ASes; and
-- down-segments, which allow remote entities to reach this AS.
+## Intra-ISD Path Segment Registration {#intra-reg}
 
-The up- and down-segments do not have to be equal. An AS may want to communicate with core ASes via one or more up-segments that differ from the down-segment(s) through which it wants to be reached. Therefore, an AS can define different selection policies for the up- and down-segment sets. Also, the processes of transforming a PCB in an up-segment or a down-segment differ slightly. Both processes are described below.
+Every *registration period* (determined by each AS), the AS's Control Service selects two sets of PCBs to transform into two types of path segments:
+
+- Up segments, which allow the infrastructure entities and endpoints in this AS to communicate with core ASes; and
+- Down segments, which allow remote entities to reach this AS.
+
+The up segments and down segments do not have to be equal as AS may want to communicate with core ASes via one or more up segments that differ from the down segment(s) through which it wants to be reached. Therefore, an AS can define different selection policies for the up segment and down segment sets. In addition, the processes of transforming a PCB in an up segment or a down segment differ slightly.
```

Comment:
"as AS may want to" -> "as an AS may want to"

Reply:
Okay